lib/oppen/print_stack.rb

```diff
@@ -245,6 +250,19 @@ def write(obj)
       buffer.write(obj.to_s)
     end
 
+    # Erase the last `count` characters.
+    #
+    # @param count [Integer]
+    #
+    # @return [Nil]
+    def erase(count = 0)
+      return if !count
+      raise ArgumentError "count = #{count} should be positive" if count.negative?
```

Comment:
This looks like a syntax error to me: don't you need a comma after the exception type?

Also, it clearly not "should" but "must".  And "positive" is > 0 in English.  So either "must not be negative" or "must be non-negative"